assets/dev/js/frontend/handlers/accessibility/nested-title-keyboard-handler.js

```diff
@@ -9,11 +9,15 @@ export default class NestedTitleKeyboardHandler extends Base {
 		this.focusableElementSelector = 'audio, button, canvas, details, iframe, input, select, summary, textarea, video, [accesskey], [contenteditable], [href], [tabindex]:not([tabindex="-1"])';
 	}
 
+	getWidgetNumber() {
```

Comment:
I wasn't able to avoid the widget container class, but I have doubled the selector, to allow for the option without as well.